WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsDeactivateStripeAdminView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import Yosemite
+
+struct InPersonPaymentsDeactivateStripeAdmin: View {
+    let onRefresh: () -> Void
+    @State var presentedSetupURL: URL? = nil
```

Comment:
nit: could this state var be private?
```suggestion
    @State private var presentedSetupURL: URL? = nil
```